test/test_age_server.py

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env python
```

Comment:
Should be updated with the recent file from the Python-based application, which also includes tests for metrics.